cmd/soroban-rpc/internal/config/options.go

```diff
@@ -13,11 +13,14 @@ import (
 
 	"github.com/stellar/go/network"
 	"github.com/stellar/go/support/strutils"
-
-	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/ledgerbucketwindow"
 )
 
-const defaultHTTPEndpoint = "localhost:8000"
+const (
+	// OneDayOfLedgers is (roughly) a 24 hour window of ledgers.
+	OneDayOfLedgers = 17280
```

Comment:
This constant already exists somewhere, I think?

Reply:
hmmm this was always here in options.go. I did not create the new one